sdk/samples/iot/README.md

```diff
@@ -138,6 +138,7 @@ To run the samples, ensure you have the following programs or tools installed on
     ```
 
   - For non-Linux based systems, download the [OpenSSL 1.1.1 LTS](https://www.openssl.org/source/openssl-1.1.1g.tar.gz) command line utility and follow the downloaded INSTALL document.
```

Comment:
Nit: What happens if OpenSSL 1.1.1 is deprecated because whatever?  Unlike PS1 where you should always be fine on latest as they're pretty good on compat, I actually don't know if we should link to 1.1.1 no matter what or latest for fear of latest breaking something?

Reply:
Also even though openssl last time I checked doesn't hand out binaries, a lot of customers might go to a site they trust anyway (though MS should NOT be in business of suggesting sites).  We might consider being a bit more generic on instructions and letting customer find this on their own since there's so many ways/versions that could change.

Reply:
We could change this to the generic https://www.openssl.org/source/ and then say 1.1.1 is recommended.  I had trouble with the most recent 3.0.0-alpha6 and found 1.1.1 worked.

Reply:
Technically the API's we are using won't work with the latest (I believe they are deprecated). So here I think we should specify that this LTS is required and we would have to revisit if a new one comes out (someone might open a GH issue saying "Hey please update to the latest).